src/pages/SongPage.tsx

```diff
@@ -59,6 +62,41 @@ const SongPage: React.FC = () => {
       console.error(e);
     }
   }
+
+  function PrevButton(songNumber: number) {
+    if (songNumber > 1) {
+      return (
+        <IonFab vertical="center" horizontal="start" slot="fixed">
+          <IonFabButton
+            color="medium"
+            onClick={() => {
+              history.push(`/${bookId}/${songNumber - 1}`);
+            }}
+          >
+            <IonIcon id="prevButton" icon={arrowBackCircleOutline} />
+          </IonFabButton>
+        </IonFab>
+      );
+    }
+  }
+
+  function NextButton(songNumber: number) {
+    // idk how to get total number of songs so its hard coded for shl for now
+    if (songNumber < 533) {
```

Comment:
One solution I think we can do is to add a field `numSongs` to the Songbook objects we have now. This way it will be dynamic, and customizable per song book.

If you wanna do it like that can you add it `numSongs` as a field in this object https://github.com/Church-Life-Apps/Songs/blob/master/src/utils/SongUtils.tsx#L46


And then directly to the JSONs of the objects in these two places?
https://github.com/Church-Life-Apps/Songs/blob/master/src/utils/SongUtils.tsx#L57-L64
https://github.com/Church-Life-Apps/Resources/blob/master/resources/songbooks.json#L2

Then you can get the songbook and check its numSongs by getting something like this:
https://github.com/Church-Life-
Apps/Songs/blob/master/src/components/MusicView.tsx#L41-L45
https://github.com/Church-Life-Apps/Songs/blob/master/src/components/MusicView.tsx#L28

Reply:
Ah actually I maybe just realized that this is a stupid idea, you can probably just pass in size of the song book array as a prop into this class. The size of the array is known from the songs list view that wraps this view

Reply:
> Looks pretty good overall! Can you check how it looks on mobile? an android phone would be best but you can also do it in chrome with the console debug settings. I'm just a little worried that it's gonna shrink the song image or overlap the words.

depending on how big your screen is, the mobile web version can get ugly pretty quickly. (maybe we could also just disable buttons and do something else for mobile web)

<img width="725" alt="mobile1" src="https://user-images.githubusercontent.com/76019114/147914025-d4869080-15a5-4344-ad89-90bea286f3af.png">
<img width="527" alt="mobile2" src="https://user-images.githubusercontent.com/76019114/147914158-480d9f11-bbcf-4bcf-81f8-11e6e76d788a.png">
<img width="584" alt="mobile3" src="https://user-images.githubusercontent.com/76019114/147913911-81a30b40-e211-4120-a93a-f8111e60ccdc.png">
<img width="423" alt="mobile4" src="https://user-images.githubusercontent.com/76019114/147913918-e270b7c0-faa3-40e5-9f21-8fb0452b1cde.png">

Reply:
ah i commented on the wrong place.

Reply:
> Ah actually I maybe just realized that this is a stupid idea, you can probably just pass in size of the song book array as a prop into this class. The size of the array is known from the songs list view that wraps this view

How does this work? Not too familiar with how props work but I thought songpage.tsx wasn't wrapped by anything? Are you saying we pass a prop into songpage.tsx? I don't think i fully understand